src/main/java/com/code_intelligence/jazzer/agent/Agent.kt

```diff
@@ -113,7 +114,7 @@ fun installInternal(
     // not be considered when resolving references to hook methods, leading to NoClassDefFoundError
     // being thrown.
     Hooks.appendHooksToBootstrapClassLoaderSearch(instrumentation, customHookNames.toSet())
-    val (includedHooks, customHooks) = Hooks.loadHooks(includedHookNames.toSet(), customHookNames.toSet())
+    val (includedHooks, customHooks) = Hooks.loadHooks(Opt.additionalClassesExcludes, includedHookNames.toSet(), customHookNames.toSet())
```

Comment:
I want to remove any Java built in classes from my output so that this works for Android. To do this we will need to skip instrumenting the additionalClasses from the hooks which are all Java built in classes, this will remove them from my output. This flag will allow me to set additional_classes_excludes=**.

Maybe a better approach would be to have a --instrument_only flag and an --instrument_only_android flag, where the --instrument_only_android flag does this for the user automatically.